src/components/LoginModal.vue

```diff
@@ -36,10 +46,113 @@
 
 <script>
 export default {
-  name: 'login-modal'
+  name: 'login-modal',
+
+  data () {
+    return {
+      username: '',
+      password: ''
+    }
+  },
+
+  computed: {
+    // function to ensure form has been filled out (used for button disable/enable)
+    completed_form: function () { return this.username && this.password }
+  },
+
+  methods: {
+
+    // function that logs in the user specified once it's called
+    async login_user () {
+      document.getElementById('error-display-login').innerText = ''
+      const getCredentials = () => {
+        const user = {
+          // email: "feathers@example.com",
+          // password: "secret"
+          username: this.username,
+          password: this.password
+        }
+        console.log(user.username)
```

Comment:
do we want this printed out later on ?

Reply:
No, I will comment them out, want to keep them for debugging the change username and password stuff later.